draft-ietf-masque-connect-ethernet.md

```diff
@@ -128,7 +128,7 @@ https://masque.example.org/?user=bob
 ~~~
 
 An implementation that supports connecting to different Ethernet segments might
-add a "vlan-identifier" variable to specify which segment to connect to. The
+add a "-identifier" variable to specify which segment to connect to. The
```

Comment:
The removal of `vlan` in this section makes it harder to understand. This is intended to be a hypothetical configuration example that uses URI template variables, which we only include because our default configuration examples do not use any URI template variables. I would leave these in, or remove the hypothetical examples altogether.